lib/iris/fileformats/grib/message.py

```diff
@@ -459,8 +459,12 @@ def _get_key_value(self, key):
             # By default these values are returned as unhelpful strings but
             # we can use int representation to compare against instead.
             res = gribapi.grib_get(self._message_id, key, int)
+            if gribapi.grib_is_missing(self._message_id, key) == 1:
```

Comment:
get_key function, with inverted order, os check, then get

add 
`# Implementation of Regulations 92.1.4 and 92.1.5 via eccodes.`
to docstring